lib/foreman_maintain/reporter/resolution_reporter.rb

```diff
@@ -0,0 +1,66 @@
+module ForemanMaintain
+  class Reporter
+    class ResolutionReporter
+      extend Forwardable
+
+      attr_reader :scenario, :status, :steps
+
+      def_delegator :cli_reporter, :puts
+
+      def initialize(cli_reporter, scenario, options = {})
+        @scenario = scenario
+        @status = options[:status]
+        @cli_reporter = cli_reporter
+      end
+
+      def cli_reporter
+        @cli_reporter
+      end
+
+      def show
+        starting_banner
+        resolution_steps
+        ending_banner
+      end
+
+      private
+
+      def type
+        if status == :error
+          'failed'
+        else
+          'warned'
+        end
+      end
+
+      def messages?
```

Comment:
By convention, methods ending in ? return a boolean. 
Instead of calling `empty?` method on `messages?` below, could you please modify method `messages?`
so that `empty?` condition will be handled by this method.

Reply:
Ah. right, fixed!

Reply:
Was this really fixed?

Reply:
My original VM got crashed. Changes overwrote because of force push on new VM(which was not updated). Fixed for real now :smile: